gnomad/utils/reference_genome.py

```diff
@@ -49,11 +49,13 @@ def get_reference_ht(
             }
         )
 
-    context = None
+    context = []
     for contig in contigs:
+        n_partitions = max(1, int(ref.contig_length(contig) / 5000000))
+        logger.info(f"Creating reference contig {contig} with {n_partitions} partitions.")
         _context = hl.utils.range_table(
             ref.contig_length(contig),
-            n_partitions=int(ref.contig_length(contig) / 5000000),
+            n_partitions=n_partitions
```

Comment:
was this creating tables with 0 partitions before you added the line of code with `max`?

Reply:
yes exactly! I got into troubles when I forgot to remove all the silly `GL*` contigs